source/meta.tex

```diff
@@ -2972,6 +2998,107 @@
 \end{codeblock}
 \end{example}
 
+\rSec2[meta.string.literal]{Checking string literals}
+
+\indexlibraryglobal{is_string_literal}%
+\begin{itemdecl}
+consteval bool is_string_literal(const char* p);
+consteval bool is_string_literal(const wchar_t* p);
+consteval bool is_string_literal(const char8_t* p);
+consteval bool is_string_literal(const char16_t* p);
+consteval bool is_string_literal(const char32_t* p);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\begin{itemize}
+\item
+  If \tcode{p} points to an unspecified object\iref{expr.const},
+  \tcode{false}.
+\item
+  Otherwise, if \tcode{p} points to a subobject
+  of a string literal object\iref{lex.string},
+  \tcode{true}.
+\item
+  Otherwise, \tcode{false}.
+\end{itemize}
+\end{itemdescr}
+
+%FIXME: weird capitalization;
+%pretty sure titles are supposed to be sentence case
```

Comment:
Yes, it shouldn't be capitalized like that. It's not necessarily a problem to repeat the title, e.g. we have loads of "General" titles.